test/integration/test_utils.js

```diff
@@ -73,3 +73,19 @@ function getComputedStyleSelector(id) {
   return `getComputedStyle(${getQuerySelector(id)})`;
 }
 exports.getComputedStyleSelector = getComputedStyleSelector;
+
+const editorPrefix = "#pdfjs_internal_editor_";
+exports.editorPrefix = editorPrefix;
+
+function getSelectedEditors(page) {
+  return page.evaluate(pref => {
```

Comment:
Nit: Can this helper function perhaps use the same parameter name as before it was moved to this file, since I find "prefix" a slightly clearer name overall (since "pref" could generally be short for e.g. preference as well)?

```suggestion
  return page.evaluate(prefix => {
```